c2rust-transpile/src/rust_ast/mod.rs

```diff
@@ -28,14 +28,6 @@ pub fn pos_to_span(pos: BytePos) -> Span {
     SpanExt::new(pos.0, pos.0)
 }
 
-pub const DUMMY_SP: Span = unsafe {
-    std::mem::transmute(SpanRepr {
-        compiler_or_fallback: 1,
-        lo: 0,
-        hi: 0,
-    })
-};
```

Comment:
`validate_repr` asserts that these are equivalent, so using the safe `Span::call_site` instead is preferrable.